engines/agos/drivers/accolade/adlib.cpp

```diff
@@ -286,19 +286,16 @@ void MidiDriver_Accolade_AdLib::setTimerCallback(void *timerParam, Common::Timer
 
 void MidiDriver_Accolade_AdLib::noteOn(byte FMvoiceChannel, byte note, byte velocity) {
 	byte adjustedNote     = note;
-	byte adjustedVelocity = velocity;
 	byte regValueA0h      = 0;
 	byte regValueB0h      = 0;
 
 	// adjust velocity
-	int16 channelVolumeAdjust = _channels[FMvoiceChannel].volumeAdjust;
-	channelVolumeAdjust += adjustedVelocity;
-	channelVolumeAdjust = CLIP<int16>(channelVolumeAdjust, 0, 0x7F);
-
+	int16 channelVolumeAdjust = velocity + _channels[FMvoiceChannel].volumeAdjust;
 	// adjust velocity with the master volume
-	byte volumeAdjust = adjustedVelocity * ((float) (128 + _masterVolume) / 128);
+	channelVolumeAdjust = (channelVolumeAdjust * (128 + _masterVolume)) / 128;
```

Comment:
I think the problem is here. _masterVolume has a range of -128 to 127, so after adding 128 0 to 255. It is then multiplied with a MIDI velocity (0-127) and then divided by 128, leaving a value in the range of 0 to 255. It should probably be divided by 255 to again get a MIDI velocity value range (0-127).